nova/conf/vmware.py

```diff
@@ -363,6 +370,15 @@
 Possible values:
 
 * Any string or empty to keep VMware default
+"""),
+    cfg.StrOpt('bigvm_deployment_free_host_hostgroup_name',
+        default='',
```

Comment:
I did actually set a default, but then the tests started failing because we don't mock the code for adding vm-groups everywhere. Easier way out was having that feature disabled by default (i.e. empty default).